src/components/Banner/Banner.less

```diff
@@ -0,0 +1,90 @@
+@banner: lucid-Banner;
+.@{banner} {
+	.box-sizing();
+	border: 1px solid @color-default-borderColor;
+	background-color: @color-default-backgroundColor;
+	font: @font;
+	border-radius: @Banner-size-borderRadiaus;
+	padding: 0 @Banner-size-spacing-rightLeft;
+	display: flex;
+
+	&.@{banner}-has-close {
```

Comment:
no because we need the `lucid-Bannder` class name.  Since two classes are being added to the elements each having the `lucid-Banner` name.